api/services/microConfig.js

```diff
@@ -177,7 +177,7 @@ function _generateImage(bag, next) {
 
   if (bag.component === 'genExec')
     bag.config.image = util.format('%s/genexec:%s',
-      'shipimg', bag.releaseVersion);
+      'drydock', bag.releaseVersion);
```

Comment:
can we not use the env value PUBLIC_IMAGE_REGISTRY here instead of hardcoding `drydock`